src/item.cpp

```diff
@@ -218,6 +219,22 @@ item& item::deactivate( const Character *ch, bool alert )
     return *this;
 }
 
+item& item::activate()
+{
+    if( active ) {
+        return *this; // no-op
```

Comment:
I assume it's for chaining.

Chaining can be useful for some specific kinds of classes, for example math-related ones. I can't think of a good use case for chaining `item`-related anything, though.